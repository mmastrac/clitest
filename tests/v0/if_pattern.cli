#!/usr/bin/env clitest --v0

$ echo $TARGET_OS
*

$ uname -a
if TARGET_OS == "linux" {
    ! Linux %{GREEDYDATA} GNU/Linux
}
if TARGET_OS == "macos" {
    ! Darwin %{GREEDYDATA}
}
if TARGET_OS == "windows" {
    choice {
        ! %{GREEDYDATA}Windows%{GREEDYDATA}
        ! MINGW%{GREEDYDATA}
    }
}
