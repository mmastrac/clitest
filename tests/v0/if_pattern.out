Running <root>/if_pattern.cli ...

<ignore>
---
<ignore>
---
✅ OK

<ignore>
---
<ignore>
---
✅ OK

<root>/if_pattern.cli PASSED
